packages/lodestar/src/api/impl/beacon/beacon.ts

```diff
@@ -46,7 +46,8 @@ export class BeaconApi implements IBeaconApi {
     };
     return {
       fork,
-      chainId: networkId
+      chainId: networkId,
+      genesisValidatorsRoot: state.genesisValidatorsRoot,
```

Comment:
We need to enhance Etth2 API `/node/fork` in order to compute domain correctly in validator `block/attestation` service.
Does it look ok, any better way to get `genesisValidatorsRoot` to the validator?

Reply:
I think this is fine for now